src/controllers/eventController.js

```diff
@@ -0,0 +1,27 @@
+const { eventService } = require('../services');
+const { throwError } = require('../utils');
+const { v4: uuidv4 } = require('uuid');
+
+const userByLikeController = async (req, res) => {
```

Comment:
```suggestion
const getLikedEventsByUser = async (req, res) => {
```
수한님 함수명 지으실 때, 영어로 의미가 잘 통해야 합니다. 문법도 지키셔야 하고 의미도 전달이 가능해야해요.
userByLike -> 라는건 좋아요를 기준으로 유저를 가져온다는 의미로 오인될 가능성이 높습니다.
우리는 결국 사용자가 좋아요한 이벤트를 가지고 오는 것이기 때문에 수정하시는 것이 좋을 것 같습니다. :)